babel.config.js

```diff
@@ -12,6 +12,9 @@ const babelConfig = {
 	plugins: [ [ '@automattic/transform-wpcalypso-async', { async: isBrowser && codeSplit } ] ],
 
 	env: {
+		production: {
+			plugins: [ 'babel-plugin-transform-react-remove-prop-types' ],
+		},
```

Comment:
Would this make sense to have in `calypso-build`?

Reply:
Potentially, but it can also be risky. We had the clean up Calypso before we could use it (🤞).

It's something to keep in mind, but I wouldn't rush to include it.